### PR TITLE
chore(docs): Add all options to example-config.json

### PR DESCRIPTION
**Pull request by arminrosu**
I used [lib/usageParams.js](https://github.com/contentful/contentful-export/blob/master/lib/usageParams.js) and the Readme as a reference.

Most options are set to default or default behaviour (e.g. `downloadAssets` is undefined by default, but there's a truthy check in the code)

### CODE REVIEW

File: example-config.json
Comment (arminrosu) [inline example-config.json:29]: This is the default limit on Records in a Community / Medium space. Hoped it would be enough for people to understand it's not the same as the API `limit` (which is set on `maxAllowedLimit`)

File: example-config.json
Comment (arminrosu) [inline example-config.json:24]: `undefined` by default, but there's a truthy check in the code, so the behaviour should be of `false`

**Comment by stefanjudis** [comment]
As explanation we discovered that [the docs link](https://www.contentful.com/developers/docs/tutorials/cli/import-and-export/) link to this example json file. In its current state it's a little light and leads to confusion. :) 

image

**Comment by ruderngespra** [comment]
Hey @arminrosu, thanks so much for this, it is a great improvement for our docs! We are currently having troubles with our test setup which we need to fix first in order to be able to merge your pr, but will do it immediately afterwards!

**Comment by arminrosu** [comment]
@ruderngespra glad to be of service.

**Comment by phoebeschmidt** [comment]
@arminrosu once you merge @ruderngespra's PR to your fork and then update this PR we plan to merge it ASAP :)

**Comment by arminrosu** [comment]
@phoebeschmidt wouldn't it be easier to just merge @ruderngespra PR into `master`? Go ahead if that works.

**Comment by phoebeschmidt** [comment]
> @phoebeschmidt wouldn't it be easier to just merge @ruderngespra PR into `master`? Go ahead if that works.

@arminrosu the lint updates are already in master but with your change we'd like to add the two separate example configs and update which one is used in the tests. You can cherry pick his commit, make the edit yourself, or merge his PR, whatever you prefer. That change is blocking this PR though. (yes we could have made this change directly to master, next time we'll have a more streamlined process).

**Comment by arminrosu** [comment]
@phoebeschmidt sorry, I thought you meant #477 not arminrosu/contentful-export/pull/1. I missed that one, 6 days old, shame on me. It's merged now. Is it ok now?

**Review by phoebeschmidt — APPROVED** [review]

**Pull request merged**
